### PR TITLE
chore(Accordion): generate default ids only once

### LINKED ISSUE #1369: Create better element ID generator
## Bug Report
Replace the use of `_.uniqueId` with a better implementation, as these are re-generated at every render, and is confusing.

### Steps
Replace the lodash function in `dialogBehavior.ts` and `Accordion.tsx` with a better implementation, that will generate a unique ID but only once.

### PR DESCRIPTION
**Pull request by silviuaavram**
Fixes https://github.com/stardust-ui/react/issues/1369

### CODE REVIEW

**Review by miroslavstastny — APPROVED** [review]

**Review by miroslavstastny — DISMISSED** [review]
Sorry, this is actually broken - you have multiple panels, you need unique ids per panel.

File: packages/react/src/components/Accordion/Accordion.tsx [hunk 1]
```diff
@@ -137,6 +137,8 @@ class Accordion extends AutoControlledComponent<WithAsProp<AccordionProps>, Acco
 
   private focusHandler: ContainerFocusHandler = null
   private itemRefs = []
+  private defaultAccordionTitleId = _.uniqueId('accordion-title-')
```
Comment (bmdalex) [inline packages/react/src/components/Accordion/Accordion.tsx:140]: nit `private readonly`, although I'm aware how @layershifter feels about modifiers 😆

**Review by bmdalex — APPROVED** [review]
👍

File: packages/react/src/components/Accordion/Accordion.tsx (hunk 1 above)
Comment (miroslavstastny) [reply packages/react/src/components/Accordion/Accordion.tsx:140]: Now imagine you have 10 accordions on a page each having 10 sections :-)

**Review by jurokapsiar — APPROVED** [review]

File: packages/react/src/components/Accordion/Accordion.tsx (hunk 1 above)
Comment (layershifter) [reply packages/react/src/components/Accordion/Accordion.tsx:140]: Visibility modifiers will be removed by lint rule on next week, no more discussions and comments around them

**Pull request merged**